docs/production-deployment/cloud/tcld/namespace.mdx

```diff
@@ -28,6 +30,57 @@ Alias: `n`
 - [tcld namespace retention](#retention)
 - [tcld namespace update-codec-server](#update-codec-server)
 
+## add-region
+
+Use `tcld namespace add-region` to add a standby region to an existing Temporal Cloud [Namespace](/namespaces).
+Enrolling a second region upgrades the Namespace to multi-region. <!--[multi-region](/cloud/multi-region)-->
+Once provisioned, the multi-region Namespace enables Temporal Cloud to start replicating Workflow Execution data from the active to the standby region and trigger failover during adverse conditions.
+
+`tcld namespace add-region`
+
+Alias: _none_
+
+:::caution
+
+To end multi-region service and end charges, contact [Temporal Support](https://support.temporal.io) directly.
```

Comment:
I think we should mention the charges in add-region, if we specify the end charges in remove region.

Reply:
Will update to something like. Charges for data replication will apply as soon as the new multi-region service is provisioned.

Reply:
Looking at this again, this is located in the add-region section of the coverage. I don't believe any change is needed.